EIPS/eip-1776.md

```diff
@@ -0,0 +1,73 @@
+---
+eip: 1776
+title: Legal Governance Token Standard
```

Comment:
```suggestion
title: Legal Governance Token Standard
description: Requires that a contract has an "owner", and that this owner has access to applicable functions so that it may easily enact the will of the DAO both on and off chain.
```